### PR TITLE
Add preview badge

### PR DESCRIPTION
**Pull request by shhsu**

### CODE REVIEW

File: src/app/common/vtabs.component.ts [hunk 1]
```diff
@@ -12,10 +13,10 @@ import {
     QueryList,
     ViewChildren,
 } from '@angular/core';
+import { FormsModule } from '@angular/forms';
 import { ActivatedRoute, Router, RouterModule } from '@angular/router';
-import { CommonModule, Location } from '@angular/common';
+import { BadgeModule } from '@microsoft/windows-admin-center-sdk/angular';
```
Comment (jhkimnew) [inline src/app/common/vtabs.component.ts:18]: this is not required.

File: src/app/common/vtabs.component.ts [hunk 2]
```diff
@@ -364,6 +437,8 @@ export const TABS: any[] = [
         CommonModule,
         DynamicModule,
         TitlesModule,
+        BadgeModule,
```
Comment (jhkimnew) [inline src/app/common/vtabs.component.ts:440]: BadgeModule is not required

**Review by jhkimnew — APPROVED** [review]
Reminding...  
Put some commit comment that we should revert this commit later to remove the badge.

File: src/app/common/vtabs.component.ts (hunk 1 above)
Comment (shhsu) [reply src/app/common/vtabs.component.ts:18]: fixed

File: src/app/common/vtabs.component.ts (hunk 2 above)
Comment (shhsu) [reply src/app/common/vtabs.component.ts:440]: fixed

**Pull request merged**